packages/fhi-designsystem/src/components/fhi-radio/fhi-radio.component.ts

```diff
@@ -0,0 +1,381 @@
+      );
+
+      --color-radio-outline-error: var(--fhi-color-danger-surface-hover);
+
+      /* Disabled state */
+      --opacity-disabled: var(--fhi-opacity-disabled);
+    }
+
+    :host {
+      display: flex;
+      align-items: center;
+      width: max-content;
+
+      #radio-container {
+        display: flex;
+        position: relative;
+      }
+
+      label {
+        padding-left: var(--dimension-radio-label-gap);
+        color: var(--color-radio-label);
+        font-family: var(--typography-radio-label-font-family);
+        font-size: var(--typography-radio-label-font-size);
+        font-weight: var(--typography-radio-label-font-weight);
+        line-height: var(--typography-radio-label-line-height);
+        letter-spacing: var(--typography-radio-label-letter-spacing);
+      }
+
+      input {
+        appearance: none;
+        margin: var(--dimension-radio-margin);
+        width: var(--dimension-radio-width);
+        height: var(--dimension-radio-height);
+        background-color: var(--color-radio-background);
+        border: var(--dimension-radio-border-width) solid
+          var(--color-radio-border);
+        border-radius: var(--fhi-border-radius-full);
+        transition: all var(--motion-radio-transition);
+      }
+
+      .radio-dot {
+        visibility: hidden;
+        opacity: 0;
+        transition: opacity var(--motion-radio-transition);
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+      }
+
+      input:checked {
+        border-width: var(--dimension-radio-border-width-checked);
+        ~ .radio-dot {
+          circle {
+            fill: var(--color-radio-border);
+          }
+          visibility: visible;
+          opacity: 1;
+        }
+      }
+    }
+
+    :host([disabled]) {
+      opacity: var(--opacity-disabled);
+    }
+
+    :host(:not(:disabled)) {
+      input:hover:not(:checked) {
+        border-color: var(--color-radio-border-hover);
+        background-color: var(--color-radio-background-hover);
+      }
+
+      input:active:not(:checked) {
+        background-color: var(--color-radio-background-active);
+        outline: var(--fhi-dimension-border-width-focus) solid
+          var(--color-radio-outline);
+      }
+    }
+
+    :host([status='error']) {
+      input {
+        border-color: var(--color-radio-border-error);
+        background-color: var(--color-radio-background-error);
+      }
+
+      input:checked {
+        ~ .radio-dot {
+          circle {
+            fill: var(--color-radio-border-error);
+          }
+        }
+      }
+    }
+
+    :host([status='error']:not(:disabled)) {
+      input:hover:not(:checked) {
+        border-color: var(--color-radio-border-error-hover);
+        background-color: var(--color-radio-background-error-hover);
+      }
+
+      input:active:not(:checked) {
+        background-color: var(--color-radio-background-error-active);
```

Comment:
```suggestion
        border-color: var(--color-radio-border-error-hover);
        background-color: var(--color-radio-background-error-active);
```

Beholder bolder-strong også ved eventuell mellomromstrykking.